src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1173,7 +1173,8 @@ Glyphicon for success status messages
 .alert-success:before {
     font-family: 'Glyphicons Halflings';
     display: inline-block;
-    padding-right: 0.5em;
+    padding-top: .2em;
```

Comment:
@whipermr5 So most of the suggestions in the answers to that StackOverFlow question hinge on wrapper divs for the floated div. However, here, the floated div itself is a `alert-success:before` CSS pseudo-element. Is it possible to have a wrapper div for a pseudo-element?